grid-demo/src/app/angular/row-dom-performance/custom-clr-virtual-rows.directive.ts

```diff
@@ -112,6 +112,8 @@ export class CustomClrVirtualRowsDirective<T> implements OnInit, DoCheck, OnDest
     this.updateFixedSizeVirtualScrollInputs();
   }
 
+  @Input('customClrKeyboardScrollPageSize') keyboardScrollPageSize = 10;
```

Comment:
Structural directive inputs should be prefixed with the directive name. So this one should be `customClrVirtualRowsKeyboardScrollPageSize`.